src/screens/Login/Login.js

```diff
@@ -0,0 +1,53 @@
+import React, {Component} from 'react';
+import {Text, View, ScrollView, Image, TextInput} from 'react-native';
+import CustomButton from 'src/components/CustomButton';
+import styles from './styles';
+
+class Login extends Component {
+  state = { login: '', password: '' };
+
+  static navigationOptions = { header: null };
+
+  onSubmit = () => {
+    const {login, password} = this.state;
+    !!login || !!password ? this.props.navigation.navigate('ProductsList') : alert('You shoul enter login and password');
```

Comment:
Don't use `alert` - it lead to warnings. Instead use Alert from react-native. It almost the same but native. 

[Alert](https://facebook.github.io/react-native/docs/alert#docsNav)